common/static/common/templates/discussion/templates.underscore

```diff
@@ -490,6 +490,10 @@
         </div>
         <% } %>
     </div>
+    <div class="alert alert-warning forum-warning-message" role="alert" >
+        <i class="fa fa-exclamation-triangle mt-1" aria-hidden="true"></i>
```

Comment:
can we add `mb-1` to `.post-options` element above? in that case we do not have to define `mt-1` to this element.